src/profiles/y_profile_admin_classes.prog.abap

```diff
@@ -558,6 +560,8 @@ CLASS lcl_util IMPLEMENTATION.
                                             header_text = 'Apply on Productive Code'(050) ).
         checks_tree->set_field_header_text( fieldname   = 'APPLY_ON_TESTCODE'
                                             header_text = 'Apply on Testcode'(034) ).
+        checks_tree->set_field_header_text( fieldname   = 'IGNORE_PSEUDO_COMMENTS'
+                                            header_text = 'Allow Exemptions' ).
```

Comment:
Should we add a comment here describing why this workaround was required?

Reply:
You should comment with `"` instead of `*`, and align the comment in the same position as the code it comments (based on the style guide). Besides, `*` is used for disabling codes by Eclipse.